docs/docs/dev_docs/contracts/portals/inbox.md

```diff
@@ -21,7 +22,8 @@ function sendL2Message(
 
 | Name           | Type    | Description |
 | -------------- | ------- | ----------- |
-| Recipient      | `L2Actor` | The recipient of  the message. This **MUST** match the rollup version and an Aztec contract that is **attached** to the contract making this call. If the recipient is not attached to the caller, the message cannot be consumed by it. |
+| _recipientAddress | `bytes32` | The address of the recipient. This is the address of the Aztec contract on L2. This **MUST** match the Aztec contract that is attached to the contract making the call. If the recipient is not attached to the caller, the message cannot be consumed by it. |
+| _recipientVersion | `uint256` | The version of the recipient. This is the version of the rollup that the recipient lives on. |
```

Comment:
`The version of the recipient.` reads wrongly. I would delete it and just keep the second statement `This is the version of the rollup that the recipient lives on.`

FWIW, I think using "Aztec" instead of "rollup" is easier to read and has lesser jargon